internal/kafka/internal/handlers/data_plane_cluster.go

```diff
@@ -139,9 +139,13 @@ func (h *dataPlaneClusterHandler) validateResizeInfo(request *private.DataPlaneC
 		if resizeInfo.Delta.DataRetentionSize == nil {
 			return errors.FieldValidationError("resizeInfo delta data retention size must be set")
 		}
-		if resizeInfo.Delta.IngressEgressThroughputPerSec == nil {
-			return errors.FieldValidationError("resizeInfo delta ingressegress throughput per second must be set")
+		if resizeInfo.Delta.IngressThroughputPerSec == nil {
+			return errors.FieldValidationError("resizeInfo delta ingress throughput per second must be set")
 		}
+		if resizeInfo.Delta.EgressThroughputPerSec == nil {
+			return errors.FieldValidationError("resizeInfo delta egress throughput per second must be set")
+		}
```

Comment:
Looking at the latest epic brief on capacity reporting, the resizeInfo field will be refactored and likely that `.Delta` field won't be there.
I am still swimming around the subject (I am yet to confirm it with fleetshard team)but chances are this will change to something differently.